tasks/main.yml

```diff
@@ -32,6 +32,26 @@
   changed_when: False
   failed_when: False
 
+- name: singularity | remove older Go installation
```

Comment:
I should be interesting to add a little switch as a variable if some infra want to keep for some reason all the version of GO ?

Reply:
Done